index.html

```diff
@@ -9,84 +9,63 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/css/bootstrap.min.css"
       rel="stylesheet"
     />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
     <link
       rel="stylesheet"
       href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css"
     />
     <title>Portfolio</title>
 </head>
 <body>
-    <section>
-        <nav id="navbar" class="navbar fixed-top navbar-expand-lg navbar-dark bg-secondary">
-          <div class="container-fluid">
-            <a class="navbar-brand" href="#">John Doe</a>
-            <button class="navbar-toggler" 
-              type="button" 
-              data-bs-toggle="collapse" 
-              data-bs-target="#navbarNavAltMarkup"
-              aria-controls="navbarNavAltMarkup"
-              aria-expanded="false"
-              aria-label="Toggle navigation"
-            >
-              <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
-              <div class="navbar-nav">
-                <a class="nav-link text-white" href="#portfolio">Portfolio</a>
-                <a class="nav-link text-white" href="#about">About</a>
-                <a class="nav-link text-white" href="#contact">Contact</a>
-                <i class="bi bi-envelope text-white"></i>
-              </div>
-            </div>
-          </div>
-        </nav>
-    </section>
+    <header class="top-nav">
+      <div class="topnav">
+        <a href="#home" class="active">John Doe</a>
+        <div class="my-link">
+        </div>
+        <a href="javascript:void(0);" class="icon">
+          <i class="fa fa-bars"></i>
+        </a>
```

Comment:
- [ ] Please know that this item should be wrapped inside a `nav` tag as it will serve as your navbar later, kindly fix that 😁😁